.github/workflows/wolfictl-update-gh.yaml

```diff
@@ -0,0 +1,24 @@
+name: Wolfictl Update From GitHub
+
+on:
+  # Triggers the workflow every hour
+  schedule:
+    - cron: "0 * * * *"
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  lint:
+    name: Wolfictl Update
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: wolfi-dev/wolfictl/.github/actions@main
+    - run: |
+        git config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"
```

Comment:
Where does this email address value come from?

Reply:
it means we get a nice github actions bot name making the PR...

<img width="956" alt="Screenshot 2022-12-19 at 21 52 42" src="https://user-images.githubusercontent.com/1457180/208707206-6c74668f-c57c-49a4-9ce9-da84d9dd6d34.png">

Reply:
Okay nice! May be worth a comment to explain the value, but consider that just a nit from me